tagstudio/src/core/library.py

```diff
@@ -501,8 +502,9 @@ def open_library(self, path: str | Path) -> int:
 
                     # Load Extension Blacklist ---------------------------------
                     if "ignored_extensions" in json_dump.keys():
-                        self.ignored_extensions = json_dump["ignored_extensions"]
-
+                        self.ext_list = json_dump["ignored_extensions"]
+                    if "ignore_extensions" in json_dump.keys():
```

Comment:
This variable name is very similar to the variable `ignored_extensions`, it would be better to use a different name which can be easily told apart.

btw checking for item's presence in dictionary in ...uh ...Pythonic (I hate that word) way is done this way:
`if "ignore_extensions" in json_dump:` instead of:
`if "ignore_extensions" in json_dump.keys():`

Reply:
Hi, thanks for the review. I agree that ignore_extensions is not the best name for the variable do you think use_as_blacklist would be better? I also think i am gonna rename the ignored_extensions in the json file to ext_list. 

The if statment i will change. I will also change the other ones that use the same syntax ( they are in the next lines )

Reply:
> There is some recent movement to avoid terminologies like master/slave or whitelist/blacklist, so I would avoid this term.
Oh makes sense sorry didn't think about that.

Reply:
what do you think of `use_as_denylist` ?

Reply:
I think "include list" and "exclude list" would work well here

Reply:
yep sound even better.